watcher/lib/influxdb/influxdb.go

```diff
@@ -73,7 +83,8 @@ func (c *Client) Setup() error {
 	for _, query := range queries {
 		log.Infof("%v", query)
 
-		if err := c.execQuery(query); err != nil {
+		err := c.execQuery(query)
+		if err != nil && !strings.Contains(err.Error(), "retention policy already exists") {
```

Comment:
I have a small suggestion here: instead of checking for substring here, I would convert the error to a proper trace error (`trace.AlreadyExists`) inside `c.execQuery` method, then here you'd just do `if err != nil && !trace.IsAlreadyExists(err)` as usual. Also, I would log "already exists" case.